API/Controllers/PlayersController.cs

```diff
@@ -96,4 +96,22 @@ public async Task<IActionResult> ListAdminNotesAsync(int id)
 
         return Ok(await adminNoteService.ListAsync<PlayerAdminNote>(id));
     }
+
+    [HttpGet("{id:int}/friends")]
+    [Authorize(Roles = $"{OtrClaims.Roles.User}, {OtrClaims.Roles.Client}")]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType<IEnumerable<PlayerCompactDTO>>(StatusCodes.Status200OK)]
+    public async Task<IActionResult> ListFriendsAsync(int id)
+    {
+        if (!await playerService.ExistsAsync(id))
+        {
+            return NotFound();
+        }
+
+        IEnumerable<PlayerCompactDTO>? result = await playerService.GetFriendsAsync(id);
+
+        return result is null
+            ? NotFound()
+            : Ok(result);
+    }
```

Comment:
I think this needs to live under the `UsersController` actually. Friends are only populated when a `User` interacts with the website. Thoughts? @myssto

Reply:
I relied on the issue description, so I've done it this way. Imo it should be sticked to users too

Reply:
Yeah that issue is quite old, sorry.

Reply:
Yes, this endpoint should be on the `UsersController` instead

Reply:
moved